### ISSUE TITLE
Check the permissions of the output directory and try again.

### ISSUE THREAD

**Issue by snajdovski** [ThisIsLibra/AndroidProjectCreator#12]
I tried as admin I have the same issue
This is mypath 
java -jar AndroidProjectCreator-1.3-stable-jar-with-dependencies.jar -decompile fernflower decompile.apk "C:\Users\Stefan Najdovski\Desktop\decompile"

After backsmaling apk it throws an error for the output file:
```
C:\Users\Stefan Najdovski\Desktop\library\apktool>
[+]Decompilation finished
[+]Decompling JAR with DEX2JAR


[+]An error has occurred, therefore AndroidProjectCreator has shut down. The error message is given below.

Something went wrong with the I/O during the decompilation. Check the permissions of the output directory and try again.

[+]For additional details, see the stack traces of the used tools above.
```

**Comment by ThisIsLibra** [comment]
Heya, as discussed via Twitter DMs before, I cannot help you with the information you've provided. The issues seems to be at your end, but I cannot troubleshoot through pictures that are taken via mobile phone from a laptop screen in a tilted angle. 

As such, this topic is closed.

**Comment by OrDuan** [comment]
I had the same issue, turns out I had to run the installation (`-install` flag) before - totally my fault, but maybe a friendly message could be helpful here. Thanks!

**Comment by snajdovski** [comment]
Still the same Issue on tried on macOS

**Comment by ThisIsLibra** [comment]
> Still the same Issue tried on macOS
> Tried with sudo also same output doesn't have permissions

You contacted me via Twitter, as well as posting this here. You do not provide me the information I'd need to help you out, you dont provide me all the answers I need when I ask for them. You are unhappy when I closed the ticket, yet you expect me to read your mind when it comes to helping you out. This project is made and maintained in my spare time, as is me helping other people. I kindly ask you to review your behaviour for the future.

To stay on topic: I cannot help you with the limited information that you have provided me with. Issue stays closed.

**Comment by ThisIsLibra** [comment]
Further communication happened via Twitter DMs and resulted in finding a bug related to spaces in paths. This issue has been addressed in version 1.3.1-stable. Issue permanently closed now as all issues have been resolved.
